asterius/rts/rts.gc.mjs

```diff
@@ -32,22 +35,72 @@ export class GC {
     for (const p of export_stableptrs) this.stablePtrManager.newStablePtr(p);
     this.symbolTable = symbol_table;
     this.reentrancyGuard = reentrancy_guard;
+    /**
+     * 'Yolo' mode disables garbage collection altogether
+     * (see {@link GC#performGC})
+     * @name GC#yolo
+     */
     this.yolo = yolo;
-    this.gcThreshold = gc_threshold;
+    /**
+     * Garbage collection will not be performed when the
+     * current number of MBlocks is less than {@link GC#gcThreshold}
+     * (see {link GC#performGC}).
+     * @name GC#gcThreshold
+     * @default 64
+     */
+    this.gcThreshold = gcThreshold;
+    /**
+     * Map used during evacation, in order to store
+     * the forwarding pointers from original objects to their copies
+     * (see {@link GC#evacuateClosure}).
+     * @name GC#closureIndirects
+     */
     this.closureIndirects = new Map();
+    /**
+     * Set containing the objects in the to-space
+     * @name GC#liveMBlocks
+     */
     this.liveMBlocks = new Set();
+    /**
+     * Set containing the object in the from-space
```

Comment:
There are pinned and unpinned objects in the fromspace. Only the unpinned ones gets moved and have their original mblocks marked as dead, and the pinned ones' original mblocks are still live. Worth mentioning here.